ecosystem.rst

```diff
@@ -9,6 +9,8 @@ Community Tutorials
 * `PostgREST + PostGIS API tutorial in 5 minutes <https://gis-ops.com/postgrest-postgis-api-tutorial-in-5-minutes>`_ -
   In this tutorial, GIS • OPS shows how to perform PostGIS calculations through PostgREST :ref:`s_procs` interface.
 
+* `"CodeLess" backend using postgres, postgrest and oauth2 authentication with keycloak <https://www.mathieupassenaud.fr/codeless_backend/>` 
```

Comment:
Cool!

To make it render properly, add an underscore(_) after the last backtick: 
```suggestion
* `"CodeLess" backend using postgres, postgrest and oauth2 authentication with keycloak <https://www.mathieupassenaud.fr/codeless_backend/>`_
```

Reply:
It would also be nice to add a link to the github repo https://github.com/mathieupassenaud/codeless-backend.

You could add it here in the same entry you've added or maybe inside your blog post.